plugins/hypervisors/xenserver/src/com/cloud/hypervisor/xenserver/resource/Xenserver625Resource.java

```diff
@@ -55,10 +55,10 @@ public void umountSnapshotDir(final Connection conn, final Long dcId) {}
 
     @Override
     public boolean setupServer(final Connection conn,final Host host) {
-        final com.trilead.ssh2.Connection sshConnection = new com.trilead.ssh2.Connection(_host.getIp(), 22);
+        final com.trilead.ssh2.Connection sshConnection = new com.trilead.ssh2.Connection(getXenServerHost().getIp(), 22);
         try {
             sshConnection.connect(null, 60000, 60000);
-            if (!sshConnection.authenticateWithPassword(_username, _password.peek())) {
+            if (!sshConnection.authenticateWithPassword(getData().getUsername(), getData().getPasswords().peek())) {
```

Comment:
why do you want to avoid it? I don't mind creating a local variable but it's only use would be for debugging and as this is a data object that seems senseless.